source/SpinalHDL/Design errors/scope_violation.rst

```diff
@@ -5,12 +5,12 @@ Scope violation
 Introduction
 ------------
 
-SpinalHDL will check that there no signals assigned outside it's declaration scope. This error isn't easy to trigger as it require some specific meta hardware description tricks.
+SpinalHDL will check that there no signals assigned outside the scope they are defined in. This error isn't easy to trigger as it requires some specific meta hardware description tricks.
```

Comment:
`SpinalHDL will check that there no signals assigned`
I think i spotted another error here, should it be `there are` instead of `there` alone?